tests/backup_helper.go

```diff
@@ -884,6 +899,19 @@ func GetNamespacesFromVMs(vms []kubevirtv1.VirtualMachine) []string {
 	return namespaces
 }
 
+// GetNamespacesToVMsMap gets a map of namespaces to VM names
+func GetNamespacesToVMsMap(vms []kubevirtv1.VirtualMachine) map[string][]string {
+	namespacesToVMsMap := make(map[string][]string)
+	for _, v := range vms {
+		if _, ok := namespacesToVMsMap[v.Namespace]; !ok {
+			namespacesToVMsMap[v.Namespace] = make([]string, 0)
+		}
```

Comment:
I think there is no need to initialize slices; append will do the initialization if required.

Reply:
Done